docs/modules/servers/pages/distributed/operate/metrics.adoc

```diff
@@ -11,6 +11,49 @@ You can also export the metrics to ElasticSearch and visualize them with https:/
 See xref:distributed/configure/elasticsearch.adoc#_exporting_metrics_directly_to_elasticsearch[elaticseach.properties]
 documentation for more details on how to set this up.
 
+If you prefer Prometheus, metrics can also be exposed over HTTP and made available by using **extensions.routes** in https://github.com/apache/james-project/blob/master/docs/modules/servers/pages/distributed/configure/webadmin.adoc[webadmin.properties] file:
```

Comment:
The prometheus details needs needs to be in the prometheus section (below).

Here we likely just should have a link to the below session.